integrations/argocd/examples/presynchook-webhook.yml

```diff
@@ -0,0 +1,56 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+ generateName: terrascan-hook-
+ namespace: test
+ annotations:
+   argocd.argoproj.io/hook: PreSync            
+spec:
+ ttlSecondsAfterFinished: 3600
+ template:
+   spec:
+     securityContext:
+       seccompProfile:
+         type: RuntimeDefault
+     containers:
+     - name: terrascan-argocd
+       image: <CUSTOME PRESYNC HOOK IMAGE FROM>
+       resources:
+         requests:
+           cpu: "1"
+           memory: "256Mi"
+         limits:
+           cpu: "1"
+           memory: "256Mi"
+       env:
+         - name: SERVICE_NAME
+           value: terra-controller-service.default.svc
```

Comment:
this value would be name of the service used to expose admission controller webhook.

Reply:
adding a comment in the doc to expose the admission webhook as a service would make sense